src/style.css

```diff
@@ -1,3 +1,88 @@
+:root {
+  --header-bg-color: #fff;
+  --text-color-main: #000;
+  --text-color-second: #00acdc;
+  --page-bg-color: #E5E5E5;
+  --header-height: 60px;
+}
+
+html {
+  background-color: var(--page-bg-color);
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
 body {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  height: var(--header-height);
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+.header__logo,
+.header__logo-link {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+
+.nav__list-item {
+  position: relative;
+  display: flex;
+
+  list-style-type: none;
+  margin-left: 20px;
+  height: var(--header-height);
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.nav__link:link,
+.nav__link:visited {
+  color: var(--text-color-main);
+}
+
+.nav__link:hover {
```

Comment:
You can combine this style and the next one into a single statement, as they have the same set of styles:
```
.nav__link:hover,
.nav__link.is-active {
  color: var(--text-color-second);
}
```